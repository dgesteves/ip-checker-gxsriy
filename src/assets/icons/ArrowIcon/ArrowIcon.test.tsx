import { render } from "@testing-library/react";
import Index from "./index";

describe("Index icon", () => {
  it("should render correctly", () => {
    const { container } = render(<Index />);
    expect(container).toBeInTheDocument();
  });

  it("should have the correct svg", () => {
    const { container } = render(<Index />);
    expect(container).toMatchSnapshot();
  });
});
