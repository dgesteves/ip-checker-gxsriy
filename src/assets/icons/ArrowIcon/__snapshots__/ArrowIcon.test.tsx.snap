// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index icon should have the correct svg 1`] = `
<div>
  <svg
    height="14"
    width="11"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="m2 1 6 6-6 6"
      fill="none"
      stroke="#fff"
      stroke-width="3"
    />
  </svg>
</div>
`;
